static/app/actionCreators/dashboards.tsx

```diff
@@ -3,9 +3,36 @@ import omit from 'lodash/omit';
 import {addErrorMessage} from 'sentry/actionCreators/indicator';
 import {Client} from 'sentry/api';
 import {t} from 'sentry/locale';
-import {DashboardDetails, Widget} from 'sentry/views/dashboardsV2/types';
+import {
+  DashboardDetails,
+  DashboardListItem,
+  Widget,
+} from 'sentry/views/dashboardsV2/types';
 import {flattenErrors} from 'sentry/views/dashboardsV2/utils';
 
+export function fetchDashboards(api: Client, orgSlug: string) {
```

Comment:
should we reuse this function in the `widgetBuilder` too?

Reply:
I'm thinking about reusing this there as well, but there's some extra logic around error handling and setting state that didn't easily transfer over. I'll work it into the PR to save widgets